dev/test/util/helpers.ts

```diff
@@ -334,3 +334,19 @@ export function stream<T>(...elements: Array<T | Error>): Duplex {
 export function response<T>(result: T): Promise<[T, unknown, unknown]> {
   return Promise.resolve([result, undefined, undefined]);
 }
+
+export class Post {
+  constructor(readonly title: string, readonly author: string) {}
+  byline(): string {
```

Comment:
Nit: toString()?

Optional.

Reply:
I just realized that the problem with using